src/main/resources/servers/verily.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "verily",
   "clientCredentialsFile": "verily_secret.json",
+  "cloudBuildEnabled": true,
```

Comment:
Hi Melissa. I found the # could not been used as comment in Json file. I search online https://reqbin.com/json/5wzepqmt/json-comment-example#:~:text=Can%20I%20add%20a%20comment,elements%20will%20still%20be%20data. find should use key and value. 

But unfortunately, it will cause error while list the server https://screenshot.googleplex.com/PQYCTUR9mqp4DyX could not Unrecognized it. 

Maybe we could add the comments into the server.java instead?